service/src/main/java/dk/dbc/ess/service/ExternalSearchService.java

```diff
@@ -208,13 +238,22 @@ Response requestSru(String base, String queryParam, String query, Integer start,
                 .queryParam("maximumRecords", stepValue)
                 .request(MediaType.APPLICATION_XML_TYPE)
                 .buildGet();
-        return invocation.invoke();
+        log.debug("Sending request to MetaProxy...");
+        Response res =  invocation.invoke();
+        log.debug("Response from MetaProxy was: " + res);
+        return res;
     }
 
     @Timed(name = "read-response-entity")
-    SearchRetrieveResponse responseSru(Response response) throws  Exception {
-        log.trace("entering responseSru...");
-        return response.readEntity(SearchRetrieveResponse.class);
+    SearchRetrieveResponse responseSru(Response response) throws Exception {
+        try {
+            SearchRetrieveResponse res = response.readEntity(SearchRetrieveResponse.class);
+            return res;
+        }
+        catch (ProcessingException pe) {
+            log.error("Error when processing SearchRetrieveResponse: ", pe);
+            throw pe;
```

Comment:
Er du sikker på at `pe` ikke bliver logget senere? i så fald kommer der 2 stack_trace i loggen, hvilket fylder og "gemmer" relevant information.

Reply:
Desuden er jeg gladere for:
```
log.error("...{}", e.getMessage());
log.debug("...",` e);
```
Så man kun får stack-trace når man kører "debug"